### DISCUSSION TITLE
missing Scheme attributes in fields.json (SDK 1.7.1)

### DISCUSSION THREAD

**Discussion by AnnaKSZ** [OP-TED/eForms-SDK#652] [Q&A] [answered]
Hi Lab,

I have the same questions as here: https://github.com/OP-TED/eForms-SDK/discussions/173

There are many fields in fields.json (in SDK 1.7.1) without "idSchemes" or "schemeName" attributes.
When fields.json will contain the shema references needed to use the fields?
Eg.: BT-1501(n)-Contract 

Thanks your reply in advance
Anna

**Comment by YvesJo** [comment, marked as answer]
Hi,
The attribute idScheme is never used. 
The presence of attribute may be deduced taking into consideration the XML element element type and the field type. For simplification, starting SDK 1.9, we've decided to define further fields (for the attributes) and have them added to the JSON file.
Regarding the identifiers fields with their associated schemeName attributes, there is, for previous versions, information about the content in the rules set, and the schemeName value is driven by the selected value.

**Comment by AnnaKSZ** [comment]
Thanks for the reply
Anna
